x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/build_alert.test.ts

```diff
@@ -6,25 +6,24 @@
  */
 
 import {
+  ALERT_INSTANCE_ID,
```

Comment:
TODO: add a unit test to ensure that we overwrite customer-generated `kibana.*` fields (use spread to overwrite `kibana.*` fields with our own).